### DISCUSSION TITLE
Nicer ncurse interface for password prompt.

### DISCUSSION THREAD

**Discussion by LexaNz** [zbm-dev/zfsbootmenu#670] [Feature Requests]
Kia ora,

Thanks for ZBM, that an awesome software. 
I'm  new user of it, running our family PC with it. My dataset is encrypted so each boot password id asked. I'm fine with it, but to get a better family acceptance, would you think it would be  possible to get a nicer password prompt interface during 'normal' boot, not at the level of plymouth with graphical, but just a nicer ncurse centered prompt for password.

Cheers,

**Comment by zdykstra** [comment]
We're using the password prompt that the ZFS userland tooling provides. It's not on our roadmap right now to try to recreate that interaction.
